src/AppInstallerCLICore/Workflows/ShellExecuteInstallerHandler.cpp

```diff
@@ -118,6 +118,17 @@ namespace AppInstaller::CLI::Workflow
                 installerArgs += ' ' + installerSwitches.at(InstallerSwitchType::Custom);
             }
 
+            // Construct custom arg passed in by cli arg
+            if (context.Args.Contains(Execution::Args::Type::CustomSwitches))
+            {
+                std::string_view customSwitches = context.Args.GetArg(Execution::Args::Type::CustomSwitches);
+                // Since these arguments are appended to the installer at runtime, it doesn't make sense to append them if empty or whitespace
+                if (!customSwitches.empty() && !std::all_of(customSwitches.begin(), customSwitches.end(), isspace))

```

Comment:
There is an existing Utility::IsEmptyOrWhitespace() implementation in AppInstallerStrings.h, please use that.

Reply:
I knew I was missing something!